README.md

```diff
@@ -3,40 +3,76 @@
   <img width=600 src="https://user-images.githubusercontent.com/5758427/231207493-62676aea-4cb9-4bb4-92b0-20309c8a933a.png">
 </p>
 <hr/>
+
 <p align="center">
-  <a href="https://docs.zama.ai/concrete"> 📒 Read documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
+  <a href="https://docs.zama.ai/concrete"> 📒 Documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
```

Comment:
is it ok to write "📚 FHE resources" for https://github.com/zama-ai/awesome-zama; isn't it confusing with https://fhe-org.github.io/resources?